### ISSUE TITLE
Allow to run pipeline from workdir (with -s param) and not only from src dir

### ISSUE THREAD

**Issue by olegs** [JetBrains-Research/chipseq-smk-pipeline#2]

**Comment by iromeo** [comment]
Yes, it is indented behaviour. We could make it more complicated to support launching from 'non pipeline src' folder, but is it really needed? Normally I use '-s' only for a playground when I'd like to patch it a bit. Or if pipeline folder contain several pipelines.

**Comment by olegs** [comment]
That's how I use it generally. From the data folder.

**Comment by iromeo** [comment]
IMHO such usecase breaks snakemake naming conventions, but I'll check it again (not 100% sure). Normally I do vice versa: launch from src folder and use workdir as an argument.

P.S: To be honest additional tweaks are required if your pipeline dir isn't same as workdir because we are supposed to use relative names but on runtime in some cases they are relative to current snakemake file and in other cases relative to working directory

**Comment by iromeo** [comment]
Snakemake assumes that you launch it from a pipeline dir and specify `workdir` as a parameter, e.g. snakemake CLI provides option for workdir if it isn't same as pipeline dir.

**Comment by olegs** [comment]
I'd say that since Snakemake allows this execution pattern pipeline should follow it.

**Comment by iromeo** [comment]
As I said before IMHO pipeline structure recommended by snakemake will not work with such pattern, but I'll check it
